Prebuilt RPi3 ARMv6 Python & NodeJS bindings

And out of TaskCluster:
```
pi@raspberrypi:~/tmp $ LD_LIBRARY_PATH=$HOME/tmp/ds-lib/:$LD_LIBRARY_PATH python client.py ~/tmp/output_graph.pb ~/wav/LDC93S1.wav ~/tmp/alphabet.txt 
Traceback (most recent call last):
  File "client.py", line 7, in <module>
    from deepspeech.model import Model
ImportError: No module named deepspeech.model
pi@raspberrypi:~/tmp $ PATH=$HOME/.local/bin/:$PATH pip install --user --verbose https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl
Collecting deepspeech==0.0.1 from https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl
  Looking up "https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl" in the cache
  No cache entry available
  Starting new HTTPS connection (1): queue.taskcluster.net
  "GET /v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl HTTP/1.1" 303 29
  Status code 303 not in [200, 203, 300, 301]
  Looking up "https://public-artifacts.taskcluster.net/XNAanHdDSL25ZaZjUaYBTg/0/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl" in the cache
  Current age based on date: 34
  Starting new HTTPS connection (1): public-artifacts.taskcluster.net
  "GET /XNAanHdDSL25ZaZjUaYBTg/0/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl HTTP/1.1" 304 0
  Using cached https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl
  Downloading from URL https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_armv7l.whl
Requirement already satisfied: numpy in /usr/lib/python2.7/dist-packages (from deepspeech==0.0.1)
Installing collected packages: deepspeech

Successfully installed deepspeech-0.0.1
Cleaning up...
pi@raspberrypi:~/tmp $ LD_LIBRARY_PATH=$HOME/tmp/ds-lib/:$LD_LIBRARY_PATH python client.py ~/tmp/output_graph.pb ~/wav/LDC93S1.wav ~/tmp/alphabet.txt 
she had your dark suit in greasy wash water all year
```

And:
```
pi@raspberrypi:~/tmp $ LD_LIBRARY_PATH=$HOME/tmp/ds-lib/:$LD_LIBRARY_PATH nodejs client.js ~/tmp/output_graph.pb ~/wav/LDC93S1.wav ~/tmp/alphabet.txt 
module.js:471
    throw err;
    ^

Error: Cannot find module 'deepspeech'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/pi/tmp/client.js:3:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
pi@raspberrypi:~/tmp $ npm install https://queue.taskcluster.net/v1/task/XNAanHdDSL25ZaZjUaYBTg/runs/0/artifacts/public/deepspeech-0.0.1.tgz

> deepspeech@0.0.1 install /home/pi/node_modules/deepspeech
> node-pre-gyp install --fallback-to-build

[deepspeech] Success: "/home/pi/node_modules/deepspeech/lib/binding/v0.0.1/node-v48-linux-arm/deepspeech.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile
/home/pi
└─┬ deepspeech@0.0.1 
  └─┬ node-pre-gyp@0.6.38 
    ├─┬ hawk@3.1.3 
    │ ├── boom@2.10.1 
    │ ├── cryptiles@2.0.5 
    │ ├── hoek@2.16.3 
    │ └── sntp@1.0.9 
    ├─┬ mkdirp@0.5.1 
    │ └── minimist@0.0.8 
    ├─┬ nopt@4.0.1 
    │ ├── abbrev@1.1.1 
    │ └─┬ osenv@0.1.4 
    │   ├── os-homedir@1.0.2 
    │   └── os-tmpdir@1.0.2 
    ├─┬ npmlog@4.1.2 
    │ ├─┬ are-we-there-yet@1.1.4 
    │ │ └── delegates@1.0.0 
    │ ├── console-control-strings@1.1.0 
    │ ├─┬ gauge@2.7.4 
    │ │ ├── aproba@1.2.0 
    │ │ ├── has-unicode@2.0.1 
    │ │ ├── object-assign@4.1.1 
    │ │ ├── signal-exit@3.0.2 
    │ │ ├─┬ string-width@1.0.2 
    │ │ │ ├── code-point-at@1.1.0 
    │ │ │ └─┬ is-fullwidth-code-point@1.0.0 
    │ │ │   └── number-is-nan@1.0.1 
    │ │ ├─┬ strip-ansi@3.0.1 
    │ │ │ └── ansi-regex@2.1.1 
    │ │ └── wide-align@1.1.2 
    │ └── set-blocking@2.0.0 
    ├─┬ rc@1.2.2 
    │ ├── deep-extend@0.4.2 
    │ ├── ini@1.3.4 
    │ ├── minimist@1.2.0 
    │ └── strip-json-comments@2.0.1 
    ├─┬ request@2.81.0 
    │ ├── aws-sign2@0.6.0 
    │ ├── aws4@1.6.0 
    │ ├── caseless@0.12.0 
    │ ├─┬ combined-stream@1.0.5 
    │ │ └── delayed-stream@1.0.0 
    │ ├── extend@3.0.1 
    │ ├── forever-agent@0.6.1 
    │ ├─┬ form-data@2.1.4 
    │ │ └── asynckit@0.4.0 
    │ ├─┬ har-validator@4.2.1 
    │ │ ├─┬ ajv@4.11.8 
    │ │ │ ├── co@4.6.0 
    │ │ │ └─┬ json-stable-stringify@1.0.1 
    │ │ │   └── jsonify@0.0.0 
    │ │ └── har-schema@1.0.5 
    │ ├─┬ http-signature@1.1.1 
    │ │ ├── assert-plus@0.2.0 
    │ │ ├─┬ jsprim@1.4.1 
    │ │ │ ├── assert-plus@1.0.0 
    │ │ │ ├── extsprintf@1.3.0 
    │ │ │ ├── json-schema@0.2.3 
    │ │ │ └─┬ verror@1.10.0 
    │ │ │   └── assert-plus@1.0.0 
    │ │ └─┬ sshpk@1.13.1 
    │ │   ├── asn1@0.2.3 
    │ │   ├── assert-plus@1.0.0 
    │ │   ├── bcrypt-pbkdf@1.0.1 
    │ │   ├─┬ dashdash@1.14.1 
    │ │   │ └── assert-plus@1.0.0 
    │ │   ├── ecc-jsbn@0.1.1 
    │ │   ├─┬ getpass@0.1.7 
    │ │   │ └── assert-plus@1.0.0 
    │ │   ├── jsbn@0.1.1 
    │ │   └── tweetnacl@0.14.5 
    │ ├── is-typedarray@1.0.0 
    │ ├── isstream@0.1.2 
    │ ├── json-stringify-safe@5.0.1 
    │ ├─┬ mime-types@2.1.17 
    │ │ └── mime-db@1.30.0 
    │ ├── oauth-sign@0.8.2 
    │ ├── performance-now@0.2.0 
    │ ├── qs@6.4.0 
    │ ├── safe-buffer@5.1.1 
    │ ├── stringstream@0.0.5 
    │ ├─┬ tough-cookie@2.3.3 
    │ │ └── punycode@1.4.1 
    │ ├── tunnel-agent@0.6.0 
    │ └── uuid@3.1.0 
    ├─┬ rimraf@2.6.2 
    │ └─┬ glob@7.1.2 
    │   ├── fs.realpath@1.0.0 
    │   ├── inflight@1.0.6 
    │   ├─┬ minimatch@3.0.4 
    │   │ └─┬ brace-expansion@1.1.8 
    │   │   ├── balanced-match@1.0.0 
    │   │   └── concat-map@0.0.1 
    │   └── path-is-absolute@1.0.1 
    ├── semver@5.4.1 
    ├─┬ tar@2.2.1 
    │ ├── block-stream@0.0.9 
    │ ├─┬ fstream@1.0.11 
    │ │ └── graceful-fs@4.1.11 
    │ └── inherits@2.0.3 
    └─┬ tar-pack@3.4.1 
      ├─┬ debug@2.6.9 
      │ └── ms@2.0.0 
      ├── fstream-ignore@1.0.5 
      ├─┬ once@1.4.0 
      │ └── wrappy@1.0.2 
      ├─┬ readable-stream@2.3.3 
      │ ├── core-util-is@1.0.2 
      │ ├── isarray@1.0.0 
      │ ├── process-nextick-args@1.0.7 
      │ ├── string_decoder@1.0.3 
      │ └── util-deprecate@1.0.2 
      └── uid-number@0.0.6 

npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN pi No description
npm WARN pi No repository field.
npm WARN pi No README data
npm WARN pi No license field.
pi@raspberrypi:~/tmp $ LD_LIBRARY_PATH=$HOME/tmp/ds-lib/:$LD_LIBRARY_PATH nodejs client.js ~/tmp/output_graph.pb ~/wav/LDC93S1.wav ~/tmp/alphabet.txt 
she had your dark suit in greasy wash water all year
```

--- a/native_client/Makefile
+++ b/native_client/Makefile
@@ -19,15 +19,20 @@ clean: bindings-clean
 	rm -f deepspeech
 
 deepspeech: client.cc
-	$(TOOLCHAIN)$(CXX) -o deepspeech $(CFLAGS) client.cc $(LDFLAGS)
+	$(CXX) -o deepspeech $(CFLAGS) client.cc $(LDFLAGS)
 
 bindings-clean:
 	rm -rf dist temp_build deepspeech.egg-info
 	rm -f python/model_wrap.cpp python/utils_wrap.cpp python/model.py python/utils.py
 
-bindings:
-	pip install --quiet numpy wheel
-	CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" LIBS="$(LIBS)" python ./setup.py bdist_wheel
+bindings-build:
+	pip install --quiet $(PYTHON_PACKAGES) wheel
+	AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" LIBS="$(LIBS)" $(NUMPY_INCLUDE) python ./setup.py build_ext $(PYTHON_PLATFORM_NAME)
+
+bindings-package:
+	$(NUMPY_INCLUDE) python ./setup.py bdist_wheel $(PYTHON_PLATFORM_NAME)
+
+bindings: bindings-build bindings-package
 
 run: deepspeech
 	${META_LD_LIBRARY_PATH}=${TFDIR}/bazel-bin/tensorflow:${TFDIR}/bazel-bin/native_client:${${META_LD_LIBRARY_PATH}} ./deepspeech ${ARGS}

--- a/native_client/definitions.mk
+++ b/native_client/definitions.mk
@@ -2,26 +2,37 @@ NC_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 TARGET ?= host
 TFDIR  ?= $(abspath $(NC_DIR)/../../tensorflow)
-CXX    ?= c++
 PREFIX ?= /usr/local
 
 ifeq ($(TARGET),host)
-TOOLCHAIN :=
-CFLAGS    := `pkg-config --cflags sox`
-LDFLAGS   := `pkg-config --libs sox`
+TOOLCHAIN       :=
+CFLAGS          := `pkg-config --cflags sox`
+LDFLAGS         := `pkg-config --libs sox`
+PYTHON_PACKAGES := numpy
 endif
 
 ifeq ($(TARGET),rpi3)
 TOOLCHAIN   ?= ${TFDIR}/bazel-$(shell basename "${TFDIR}")/external/GccArmRpi/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/arm-linux-gnueabihf-
-RASPBIAN    ?= ../multistrap-raspbian-jessie
+RASPBIAN    ?= $(abspath $(NC_DIR)/../multistrap-raspbian-jessie)
 CFLAGS      := -isystem $(RASPBIAN)/usr/include -isystem $(RASPBIAN)/usr/include/arm-linux-gnueabihf
 LDFLAGS     := -Wl,-rpath-link,$(RASPBIAN)/lib/arm-linux-gnueabihf -Wl,-rpath-link,$(RASPBIAN)/usr/lib/arm-linux-gnueabihf/ $(RASPBIAN)/usr/lib/arm-linux-gnueabihf/libsox.so
+
+PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_version_tuple(); print(maj+'.'+min);")
+PYTHON_PACKAGES      :=
+NUMPY_INCLUDE        := NUMPY_INCLUDE=$(RASPBIAN)/usr/include/python$(PYVER)/
+PYTHON_PLATFORM_NAME := --plat-name linux_armv7l
+NODE_PLATFORM_TARGET := --target_arch=arm --target_platform=linux
 endif
 
 OS := $(shell uname -s)
 CFLAGS  += $(EXTRA_CFLAGS)
 LIBS    := -ldeepspeech -ldeepspeech_utils -ltensorflow_cc $(EXTRA_LIBS)
 LDFLAGS += -Wl,-rpath,. -L${TFDIR}/bazel-bin/tensorflow -L${TFDIR}/bazel-bin/native_client $(EXTRA_LDFLAGS) $(LIBS)
+
+AS      := $(TOOLCHAIN)as
+CC      := $(TOOLCHAIN)gcc
+CXX     := $(TOOLCHAIN)c++
+LD      := $(TOOLCHAIN)ld
 
 META_LD_LIBRARY_PATH := LD_LIBRARY_PATH
 ifeq ($(OS),Darwin)

--- a/native_client/javascript/Makefile
+++ b/native_client/javascript/Makefile
@@ -21,10 +21,10 @@ configure: deepspeech_wrap.cxx
 	$(NODE_BUILD_TOOL) configure $(NODE_BUILD_VERBOSE)
 
 build: configure deepspeech_wrap.cxx
-	CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" LIBS="$(LIBS)" $(NODE_BUILD_TOOL) $(NODE_ABI_TARGET) rebuild $(NODE_BUILD_VERBOSE)
+	AS=$(AS) CC=$(CC) CXX=$(CXX) LD=$(LD) CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" LIBS="$(LIBS)" $(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_ABI_TARGET) rebuild $(NODE_BUILD_VERBOSE)
 
 package: build
-	$(NODE_BUILD_TOOL) $(NODE_ABI_TARGET) package $(NODE_BUILD_VERBOSE)
+	$(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_ABI_TARGET) package $(NODE_BUILD_VERBOSE)
 
 npm-pack: clean
 	npm install node-pre-gyp@0.6.x

--- a/native_client/multistrap.conf
+++ b/native_client/multistrap.conf
@@ -7,8 +7,8 @@ aptsources=Raspbian
 cleanup=true
 
 [Raspbian]
-packages=libc6-dev libsox-dev
-source=http://mirrors.ocf.berkeley.edu/raspbian/raspbian/
+packages=libc6-dev libpython2.7-dev libpython3.4-dev libsox-dev python-numpy
+source=http://mirrordirector.raspbian.org/raspbian/
 keyring=raspbian-archive-keyring
 components=main
 suite=jessie

--- a/native_client/setup.py
+++ b/native_client/setup.py
@@ -4,13 +4,19 @@ from setuptools import setup, Extension
 from distutils.command.build import build
 
 import os
-import numpy
 import subprocess
 
 try:
-    numpy_include = numpy.get_include()
-except AttributeError:
-    numpy_include = numpy.get_numpy_include()
+    import numpy
+    try:
+        numpy_include = numpy.get_include()
+    except AttributeError:
+        numpy_include = numpy.get_numpy_include()
+except ImportError:
+    numpy_include = ''
+    assert 'NUMPY_INCLUDE' in os.environ
+
+numpy_include = os.getenv('NUMPY_INCLUDE', numpy_include)
 
 class BuildExtFirst(build):
     sub_commands = [('build_ext', build.has_ext_modules),

--- a/taskcluster/.shared.yml
+++ b/taskcluster/.shared.yml
@@ -1,12 +1,17 @@
 python:
   packages:
-    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev xz-utils tk-dev'
+    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
 nodejs:
   packages:
     apt: 'nodejs sox'
     prep_4: 'echo "deb http://deb.nodesource.com/node_4.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
     prep_5: 'echo "deb http://deb.nodesource.com/node_5.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
     prep_6: 'echo "deb http://deb.nodesource.com/node_6.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
+## swig -javascript requires swig >= 3.0.7 (14.04 provides 2.0.11)
+## force 3.0.10 for nodejs V8 API compatibility
+swig:
+  packages:
+    install_script: 'wget http://mirrors.kernel.org/ubuntu/pool/universe/s/swig/swig_3.0.10-1.1_amd64.deb -O /tmp/swig_3.0.10-1.1_amd64.deb && wget http://mirrors.kernel.org/ubuntu/pool/universe/s/swig/swig3.0_3.0.10-1.1_amd64.deb -O /tmp/swig3.0_3.0.10-1.1_amd64.deb && dpkg -i /tmp/swig_3.0.10-1.1_amd64.deb /tmp/swig3.0_3.0.10-1.1_amd64.deb'
 system:
   username: 'build-user'
   homedir:

--- a/taskcluster/linux-amd64-cpu-opt.yml
+++ b/taskcluster/linux-amd64-cpu-opt.yml
@@ -4,17 +4,10 @@ build:
     - "index.project.deepspeech.deepspeech.native_client.${event.head.branch}.cpu"
     - "index.project.deepspeech.deepspeech.native_client.${event.head.branch}.${event.head.sha}.cpu"
     - "index.project.deepspeech.deepspeech.native_client.cpu.${event.head.sha}"
-  ## swig -javascript requires swig >= 3.0.7 (14.04 provides 2.0.11)
-  ## force 3.0.10 for nodejs V8 API compatibility
   system_setup:
     - >
-      echo "deb http://deb.nodesource.com/node_6.x trusty main" > /etc/apt/sources.list.d/nodesource.list &&
-      wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - &&
-      apt-get -qq update && apt-get -qq -y install nodejs python-yaml &&
-      apt-get -qq -y install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev &&
-      wget http://mirrors.kernel.org/ubuntu/pool/universe/s/swig/swig_3.0.10-1.1_amd64.deb -O /tmp/swig_3.0.10-1.1_amd64.deb &&
-      wget http://mirrors.kernel.org/ubuntu/pool/universe/s/swig/swig3.0_3.0.10-1.1_amd64.deb -O /tmp/swig3.0_3.0.10-1.1_amd64.deb &&
-      dpkg -i /tmp/swig_3.0.10-1.1_amd64.deb /tmp/swig3.0_3.0.10-1.1_amd64.deb
+      ${nodejs.packages.prep_6} && apt-get -qq update && apt-get -qq -y install nodejs python-yaml &&
+      apt-get -qq -y install ${python.packages.apt} && ${swig.packages.install_script}
   tensorflow: "https://index.taskcluster.net/v1/task/project.deepspeech.tensorflow.pip.master.cpu/artifacts/public/home.tar.xz"
   summarize_graph: "https://index.taskcluster.net/v1/task/project.deepspeech.tensorflow.pip.master.cpu/artifacts/public/summarize_graph"
   scripts:

--- a/taskcluster/linux-rpi3-cpu-opt.yml
+++ b/taskcluster/linux-rpi3-cpu-opt.yml
@@ -11,7 +11,9 @@ build:
     - >
       apt-get -qq -y install gdebi git pixz &&
       wget http://mirrors.kernel.org/ubuntu/pool/universe/m/multistrap/multistrap_2.2.0ubuntu2_all.deb -O /tmp/multistrap_2.2.0ubuntu2_all.deb &&
-      echo "y" | gdebi /tmp/multistrap_2.2.0ubuntu2_all.deb
+      echo "y" | gdebi /tmp/multistrap_2.2.0ubuntu2_all.deb &&
+      ${nodejs.packages.prep_6} && apt-get -qq update && apt-get -qq -y install nodejs python-yaml &&
+      apt-get -qq -y install ${python.packages.apt} && ${swig.packages.install_script}
   system_config:
     - >
       multistrap -d /tmp/multistrap-raspbian-jessie/ -f ${system.homedir.linux}/DeepSpeech/ds/native_client/multistrap.conf

--- a/taskcluster/rpi3-build.sh
+++ b/taskcluster/rpi3-build.sh
@@ -21,3 +21,8 @@ BAZEL_BUILD_FLAGS="${BAZEL_ARM_FLAGS}"
 do_bazel_build
 
 do_deepspeech_binary_build
+
+export SUPPORTED_PYTHON_VERSIONS="2.7.13 3.4.6"
+do_deepspeech_python_build
+
+do_deepspeech_nodejs_build

--- a/tc-tests-utils.sh
+++ b/tc-tests-utils.sh
@@ -20,6 +20,11 @@ export BAZEL_CTC_TARGETS="//native_client:ctc_decoder_with_kenlm"
 
 model_name=$(basename "${DEEPSPEECH_TEST_MODEL}")
 
+SUPPORTED_PYTHON_VERSIONS=${SUPPORTED_PYTHON_VERSIONS:-2.7.13 3.4.6 3.5.3 3.6.2}
+# 7.10.0 and 8.0.0 targets fails to build
+# > ../deepspeech_wrap.cxx:966:23: error: 'WeakCallbackData' in namespace 'v8' does not name a type
+SUPPORTED_NODEJS_VERSIONS=${SUPPORTED_NODEJS_VERSIONS:-4.8.0 5.12.0 6.10.0}
+
 assert_correct_inference()
 {
   phrase=$1
@@ -188,12 +193,14 @@ do_deepspeech_python_build()
 
   mkdir -p wheels
 
-  for pyver in 2.7.13 3.4.6 3.5.3 3.6.2; do
+  for pyver in ${SUPPORTED_PYTHON_VERSIONS}; do
     pyenv install ${pyver}
     pyenv virtualenv ${pyver} deepspeech
     source ${PYENV_ROOT}/versions/${pyver}/envs/deepspeech/bin/activate
 
     EXTRA_CFLAGS="${EXTRA_LOCAL_CFLAGS}" EXTRA_LDFLAGS="${EXTRA_LOCAL_LDFLAGS}" EXTRA_LIBS="${EXTRA_LOCAL_LIBS}" make -C native_client/ \
+      TARGET=${SYSTEM_TARGET} \
+      RASPBIAN=/tmp/multistrap-raspbian-jessie \
       TFDIR=${DS_TFDIR} \
       bindings-clean bindings
 
@@ -212,10 +219,10 @@ do_deepspeech_nodejs_build()
 
   export PATH="$(npm root)/.bin/:$PATH"
 
-  # 7.10.0 and 8.0.0 targets fails to build
-  # > ../deepspeech_wrap.cxx:966:23: error: 'WeakCallbackData' in namespace 'v8' does not name a type
-  for node in 4.8.0 5.12.0 6.10.0; do
+  for node in ${SUPPORTED_NODEJS_VERSIONS}; do
     EXTRA_CFLAGS="${EXTRA_LOCAL_CFLAGS}" EXTRA_LDFLAGS="${EXTRA_LOCAL_LDFLAGS}" EXTRA_LIBS="${EXTRA_LOCAL_LIBS}" make -C native_client/javascript \
+      TARGET=${SYSTEM_TARGET} \
+      RASPBIAN=/tmp/multistrap-raspbian-jessie \
       TFDIR=${DS_TFDIR} \
       NODE_ABI_TARGET=--target=$node \
       clean package

